kola/tests/docker/docker.go

```diff
@@ -170,6 +170,17 @@ systemd:
       [Install]
       WantedBy=multi-user.target`),
 	})
+	register.Register(&register.Test{
+		// Ensure containerd gets back up when it dies
+		Name:        "docker.containerd-restart",
+		Run:         dockerContainerdRestart,
+		ClusterSize: 1,
+		UserData: conf.ContainerLinuxConfig(`
+systemd:
+  units:
+   - name: docker.service
+     enable: true`),
+	})
```

Comment:
The next-next alpha, which will be 1520.  Of course you'll need to disable that field when testing.